### PR TITLE
Create topekazoo.org.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: src/chrome/content/rules/topekazoo.org.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!--
+	Invalid certificate:
+		email.topekazoo.org
+		imap.topekazoo.org
+		mail.topekazoo.org
+		pop.topekazoo.org
+		smtp.topekazoo.org
+		store.topekazoo.org
```
Comment (J0WI) [inline src/chrome/content/rules/topekazoo.org.xml]: WFM

File: src/chrome/content/rules/topekazoo.org.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		webmail.topekazoo.org
+
+	Refused:
+		mx.topekazoo.org
```
Comment (J0WI) [inline src/chrome/content/rules/topekazoo.org.xml]: Same for plain HTTP, so this can be removed from the comment section.

Comment (TotalCaesar659) [reply src/chrome/content/rules/topekazoo.org.xml]: Do you mean that I don't need to add all these invaild domains to comment? Cause many other reviewers asked me to add some subdomains or check all ones. I'm not good in subdomain classification, and even contributing.md doesn't help me muchю

Comment (J0WI) [reply src/chrome/content/rules/topekazoo.org.xml]: If they don't run any HTTP service like `mx.topekazoo.org` you can skip them. All others with certificate issues etc. should be documented.

File: src/chrome/content/rules/topekazoo.org.xml (hunk 1 above)
Comment (TotalCaesar659) [reply src/chrome/content/rules/topekazoo.org.xml]: Invalid certificate for me, but OK, I'll move it.

**Pull request merged**